### ISSUE TITLE
I pulled this for version 4.1 and it doesnt even install

### ISSUE THREAD

**Issue by staycreativedesign** [MatthewKennedy/spree_shipstation#4]
Can you update for version 4.1?

**Comment by MatthewKennedy** [comment]
@staycreativedesign 

Not a problem, have you come across an error using this so far?

**Comment by staycreativedesign** [comment]
yes I have spree version 4.1 and it doesnt install doe shtis work with
version 4.1 ?

**Comment by MatthewKennedy** [comment]
I'll finish updating this tomorrow.

**Comment by staycreativedesign** [comment]
thanks man!

**Comment by MatthewKennedy** [comment]
@staycreativedesign you can try the new version: [https://github.com/MatthewKennedy/spree_shipstation](https://github.com/MatthewKennedy/spree_shipstation) let me know if you have any issues.
